src/simple-cache-client.ts

```diff
@@ -147,6 +148,33 @@ export class SimpleCacheClient {
     return await client.delete(cacheName, key);
   }
 
+  public async listFetch(
+    cacheName: string,
+    listName: string | Uint8Array
```

Comment:
I don't think we let users to pass dictionary/set/list names in bytes?

Reply:
+1 to just `listName: string`

Reply:
list_name in the protobuf is bytes. Are we only taking strings in the SDK?

Reply:
Oh hmm, looking at the other methods we do allow key to be `Uint8Array`. So I'm actually supportive of `string | Uint8Array`. In csharp-sdk we don't allow this, but I think it's better here to align with the the other method signatures in this sdk.

Reply:
I'm happy to take only strings and add bytes later if it's necessary.